Packs/RubrikPolaris/Integrations/RubrikPolaris/RubrikPolaris.yml

```diff
@@ -82,7 +106,27 @@ script:
     - contextPath: Rubrik.Radar.ActivitySeriesId
       description: The ID of the Rubrik Polaris Activity Series.
       type: String
-  dockerimage: demisto/python3:3.9.4.18682
+  - arguments:
+    - description: |
+        The id of the CDM cluster.  When used in combination with "Rubrik Radar Anomaly" incidents, this value will automatically be looked up using the incident context. Otherwise it is a required value.
```

Comment:
```suggestion
        The id of the CDM cluster. When used in combination with "Rubrik Radar Anomaly" incidents, this value will automatically be looked up using the incident context. Otherwise, it is a required value.
```